ui.tests/test-module/specs/telephoneinput/telephoneinput.authoring.spec.js

```diff
@@ -74,6 +74,60 @@ describe('Page - Authoring', function () {
             testTelephoneInputBehaviour(telephoneInputEditPathSelector, telephoneInputDrop);
         });
     })
+
+    it('change validation pattern type of TelephoneInput', function () {
+      const bemEditDialog = '.cmp-adaptiveform-telephoneinput__editdialog';
+      dropTelephoneInputInContainer();
+      cy.openEditableToolbar(sitesSelectors.overlays.overlay.component + telephoneInputEditPathSelector);
+      cy.invokeEditableAction("[data-action='CONFIGURE']");
+      cy.get(bemEditDialog).contains('Validation').click({force: true}).then(() => {
+        cy.get('.cmp-adaptiveform-telephoneinput__validationformat').should('have.value', '^[+][0-9]{0,14}$');
+        cy.get('.cmp-adaptiveform-telephoneinput__validationpattern select').select('US Phone Number', {force: true});
```

Comment:
do you need force:true here?? this can lead to inconsistency , put the next assertion in then block, this will result in inconsistency.

Reply:
This force: true is also required because the element will not be in the view